[2025-06-04T19:15:52.862+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-04T19:15:52.925+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_training.cleanup_resources scheduled__2025-06-03T03:00:00+00:00 [queued]>
[2025-06-04T19:15:52.962+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_training.cleanup_resources scheduled__2025-06-03T03:00:00+00:00 [queued]>
[2025-06-04T19:15:52.963+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-06-04T19:15:53.003+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): cleanup_resources> on 2025-06-03 03:00:00+00:00
[2025-06-04T19:15:53.014+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4699) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-04T19:15:53.018+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_training', 'cleanup_resources', 'scheduled__2025-06-03T03:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_training_dag.py', '--cfg-path', '/tmp/tmpwxc6axbr']
[2025-06-04T19:15:53.022+0000] {standard_task_runner.py:72} INFO - Started process 4700 to run task
[2025-06-04T19:15:53.023+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask cleanup_resources
[2025-06-04T19:15:53.149+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_training.cleanup_resources scheduled__2025-06-03T03:00:00+00:00 [running]> on host a2222215717b
[2025-06-04T19:15:53.359+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='datamasterylab.com' AIRFLOW_CTX_DAG_ID='fraud_detection_training' AIRFLOW_CTX_TASK_ID='cleanup_resources' AIRFLOW_CTX_EXECUTION_DATE='2025-06-03T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-03T03:00:00+00:00'
[2025-06-04T19:15:53.361+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-06-04T19:15:53.392+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-06-04T19:15:53.394+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'rm -f /app/tmp/*.pkl']
[2025-06-04T19:15:53.412+0000] {subprocess.py:99} INFO - Output:
[2025-06-04T19:15:53.427+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-06-04T19:15:53.481+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-06-04T19:15:53.482+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_training, task_id=cleanup_resources, run_id=scheduled__2025-06-03T03:00:00+00:00, execution_date=20250603T030000, start_date=20250604T191552, end_date=20250604T191553
[2025-06-04T19:15:53.561+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-04T19:15:53.599+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-04T19:15:53.603+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-06-05T01:49:20.387+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-05T01:49:20.419+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_training.cleanup_resources scheduled__2025-06-03T03:00:00+00:00 [queued]>
[2025-06-05T01:49:20.434+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_training.cleanup_resources scheduled__2025-06-03T03:00:00+00:00 [queued]>
[2025-06-05T01:49:20.435+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-06-05T01:49:20.455+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): cleanup_resources> on 2025-06-03 03:00:00+00:00
[2025-06-05T01:49:20.462+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=314) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-05T01:49:20.465+0000] {standard_task_runner.py:72} INFO - Started process 315 to run task
[2025-06-05T01:49:20.464+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_training', 'cleanup_resources', 'scheduled__2025-06-03T03:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_training_dag.py', '--cfg-path', '/tmp/tmpi8238coe']
[2025-06-05T01:49:20.466+0000] {standard_task_runner.py:105} INFO - Job 6: Subtask cleanup_resources
[2025-06-05T01:49:20.538+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_training.cleanup_resources scheduled__2025-06-03T03:00:00+00:00 [running]> on host 711694a4a9ee
[2025-06-05T01:49:20.648+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='datamasterylab.com' AIRFLOW_CTX_DAG_ID='fraud_detection_training' AIRFLOW_CTX_TASK_ID='cleanup_resources' AIRFLOW_CTX_EXECUTION_DATE='2025-06-03T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-03T03:00:00+00:00'
[2025-06-05T01:49:20.649+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-06-05T01:49:20.710+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-06-05T01:49:20.712+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'rm -f /app/tmp/*.pkl']
[2025-06-05T01:49:20.724+0000] {subprocess.py:99} INFO - Output:
[2025-06-05T01:49:20.730+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-06-05T01:49:20.763+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-06-05T01:49:20.764+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_training, task_id=cleanup_resources, run_id=scheduled__2025-06-03T03:00:00+00:00, execution_date=20250603T030000, start_date=20250605T014920, end_date=20250605T014920
[2025-06-05T01:49:20.846+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-05T01:49:20.871+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-05T01:49:20.875+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
